recipes/scriv/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scriv" %}
+{% set dunder_name = name.replace("-", "_") %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ dunder_name }}/{{ dunder_name }}-{{ version }}.tar.gz
+  sha256: 30ae9ff8d144f8e0cf394c4e1d379542f1b3823767642955b54ec40dc00b32b6
+
+build:
+  entry_points:
+    - scriv = scriv.cli:cli
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.8
```

Comment:
```suggestion
    - python >=3.7
```

Reply:
The upstream project requires 3.8 

https://github.com/nedbat/scriv/blob/16fe892e7c15f4a3a2bb104591389307c76ca2de/setup.cfg#L36C1-L36C17

Reply:
In the future version. This one is 3.7 https://github.com/nedbat/scriv/blob/1.5.1/setup.cfg#L102

Reply:
Oh right but ITHO https://conda-forge.org/news/2022/08/24/dropping-python-37/

Reply:
Sure, but that doesn't affect that this package will be, correctly installed on 3.7 if you follow the package metadata.